Common/Inc/CommonDataTypes.hpp

```diff
@@ -5,8 +5,11 @@
 #ifndef ZPSW3_COMMON_DATATYPES_HPP
 #define ZPSW3_COMMON_DATATYPES_HPP
 
+#include <stdint.h>
+
+namespace LOS {
 // Struct copied from old code
-typedef struct {
+struct LosSFData {
```

Comment:
no typedef?

Reply:
in cpp structs/enums/classes/whatever other user defined data types are implicitly typedef'ed

Reply:
I can typedef it, but our style guide doesnt mention this iirc, and our code before this common datatypes change was a mix of typdef and non typdef. I personally think its cleaner without typedef.

Reply:
closing this as implicit typdef is more c++ style, and this is c++ code.